util/storage-samples/storage-owner-sample/src/main/java/org/fido/iot/storage/OwnerDbStorage.java

```diff
@@ -267,7 +268,39 @@ public UUID getReplacementGuid() {
 
   @Override
   public Composite getReplacementOwnerKey() {
-    return getCryptoService().getOwnerPublicKey(voucher);
+    List<PublicKey> certs = null;
```

Comment:
Would it be better to return NULL/current owner key, in case no customer is assigned? That'd make assigning the next owner optional and would default to RESALE, as it does right now.
Additionally, this means that we only need to add 1 customer key by default - reseller keys, since the current owner keys are already taken care off by the voucher's key.

Reply:
Implemented as discussed.